tools/setup/emoji/build_emoji

```diff
@@ -153,6 +155,9 @@ def code_point_to_file_name_map(ttx):
 
 def main():
     # type: () -> None
+    run(["sudo", "mkdir", "-p", EMOJI_CACHE_PATH])
+    run(["sudo", "chown", "%s:%s" % (user_id, user_id), EMOJI_CACHE_PATH])
```

Comment:
On `sudo` in Travis: https://travis-ci.org/rht/zulip/builds/263741821, https://github.com/zulip/zulip/blob/6e9f8b90dcefbc8a0bc45b3e9f62b8f7f10fd2a9/.travis.yml#L69. With this, I can't understand the intent of the comment in
```
if 'TRAVIS' in os.environ:
    # In Travis CI, we don't have root access
    EMOJI_CACHE_PATH = "/home/travis/zulip-emoji-cache"
```